app/src/main/java/com/techcourse/controller/LoginController.java

```diff
@@ -4,26 +4,34 @@
 import com.techcourse.repository.InMemoryUserRepository;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-import nextstep.mvc.controller.Controller;
+import nextstep.mvc.view.JspView;
+import nextstep.mvc.view.ModelAndView;
+import nextstep.web.annotation.Controller;
+import nextstep.web.annotation.RequestMapping;
+import nextstep.web.support.RequestMethod;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-public class LoginController implements Controller {
+@Controller
+public class LoginController {
 
     private static final Logger log = LoggerFactory.getLogger(LoginController.class);
 
-    @Override
-    public String execute(final HttpServletRequest req, final HttpServletResponse res) throws Exception {
+    @RequestMapping(value = "/login", method = RequestMethod.POST)
+    public ModelAndView login(final HttpServletRequest req, final HttpServletResponse res) {
+        final String viewName;
         if (UserSession.isLoggedIn(req.getSession())) {
-            return "redirect:/index.jsp";
+            viewName = "redirect:/index.jsp";
+            return new ModelAndView(new JspView(viewName));
```

Comment:
```suggestion
            return new ModelAndView(new JspView("redirect:/index.jsp"));
```
이건 바로 반환해도 되지 않을까요?? (저의 의견일 뿐이니 반영하지 않으셔도 됩니다...!!)